examples/PlutusComparison.md

```diff
@@ -0,0 +1,288 @@
+
+We are going to compare two different off-chain contracts 
+both in Plutus and CTL. 
+
+For this discussion, we need to go through an overview of 
+both of them.
+
+
+The purpose of `Contract` in CTL is to bring the capabilities of 
+`QueryMExtended` to the public API. 
+
+The current definition of `Contract` in CTL is :
+
+```PureScript
+type QueryConfig (r :: Row Type) =
```

Comment:
I'm not sure if we need to reproduce the entire `QueryConfig` here. It will probably lead to the docs drifting away from the actual definitions. In fact, `slotConfig` has already been removed (recently)